lib/global-styles.php

```diff
@@ -273,7 +273,6 @@ function gutenberg_experimental_global_styles_get_supported_styles( $supports )
 		'--wp--style--color--link' => array( '__experimentalColor', 'linkColor' ),
 		'background-color'         => array( '__experimentalColor' ),
 		'background'               => array( '__experimentalColor', 'gradients' ),
-		'block-align'              => array( 'align' ),
```

Comment:
This was added just because of a previous implementation detail, it's not really needed here.